sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -40,49 +41,11 @@
     'K80': [1, 2, 4, 8],
 }
 
-# FIXME(woosuk): This URL can change.
-A2_PRICING_URL = '/compute/vm-instance-pricing_34568c2efd1858a89d6f5b0f1cdd171bbea1cdcba646e9771e6ef4028238086f.frame'  # pylint: disable=line-too-long
-A2_INSTANCE_TYPES = {
-    'a2-highgpu-1g': {
-        'vCPUs': 12,
-        'MemoryGiB': 85,
-    },
-    'a2-highgpu-2g': {
-        'vCPUs': 24,
-        'MemoryGiB': 170,
-    },
-    'a2-highgpu-4g': {
-        'vCPUs': 48,
-        'MemoryGiB': 340,
-    },
-    'a2-highgpu-8g': {
-        'vCPUs': 96,
-        'MemoryGiB': 680,
-    },
-    'a2-megagpu-16g': {
-        'vCPUs': 96,
-        'MemoryGiB': 1360,
-    },
-    'a2-ultragpu-1g': {
-        'vCPUs': 12,
-        'MemoryGiB': 170,
-    },
-    'a2-ultragpu-2g': {
-        'vCPUs': 24,
-        'MemoryGiB': 340,
-    },
-    'a2-ultragpu-4g': {
-        'vCPUs': 48,
-        'MemoryGiB': 680,
-    },
-    'a2-ultragpu-8g': {
-        'vCPUs': 96,
-        'MemoryGiB': 1360,
-    },
-}
-
-# Source: https://cloud.google.com/compute/docs/gpus/gpu-regions-zones
-NO_A100_16G_ZONES = ['asia-northeast3-a', 'asia-northeast3-b', 'us-west4-b']
+# Source: https://cloud.google.com/compute/docs/gpus/gpu-regions-zones#limitations
+A100_16G_ZONES = [
```

Comment:
This name is a bit confusing. I though it is A100 with 16 GB memory at the first glance. How about we rename it to `A2_MEGAGPU_16G_ZONES`? Also, it would be nice to have a comments above:
```
# A100 instance with 16 GPUs only appears in the following zones.
```

Reply:
Good point. Fixed. PTAL.